pkgs/tools/security/enum4linux-ng/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, buildPythonApplication
+, fetchFromGitHub
+, impacket
+, ldap3
+, pyyaml
+, samba
+}:
+
+buildPythonApplication rec {
+  pname = "enum4linux-ng";
+  # https://github.com/cddmp/enum4linux-ng/issues/12
+  version = "20210112-${lib.strings.substring 0 7 rev}";
```

Comment:
Not sure if you noticed, but they tagged a 1.0.0 release: https://github.com/cddmp/enum4linux-ng/releases/tag/v1.0.0.

Reply:
Yes, I tagged a release for you. Does it work for you?

Reply:
Package is now at 1.0.0